scalafix-sbt/src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -15,6 +15,8 @@ object ScalafixPlugin extends AutoPlugin {
   override def trigger: PluginTrigger = allRequirements
   override def requires: Plugins = JvmPlugin
   object autoImport {
+    val Linting = Tags.Tag("linting")
```

Comment:
`Linting` is a bit misleading considering scalafix does linting and refactoring. Would it be OK to call it `Scalafix` instead?

Reply:
the sbt manual recommends that the tag be named after the purpose of the work, not the specific implementation. I was nervous with calling it `Scalafix` incase it clashed with some other symbol of the same name... I can make the change if you're sure about it.